### DISCUSSION TITLE
Nextjs integration

### DISCUSSION THREAD

**Discussion by ShivasaiPothuganti** [wedaa-tech/wedaa-tech.github.io#9] [Ideas]
Hi, Please consider adding Nextjs, a popular react framework,  as one of the scaffolding options along with react and angular for the frontend client.

**Comment by rohith-wedaa** [comment]
Yes, we can include support for Next.js. Feel free to contribute the same as a blueprint in [Blueprints Repository](https://github.com/wedaa-tech/jhipster-blueprints).
As enterprise best practices, every framework supported by WeDAA has few standard integrations. Please create blueprint accordingly.

- OAuth (Keycloak)
- API Gateway (Spring)

Checkout React blueprint as a reference.

CC: @praveen-wedaa

**Comment by ShivasaiPothuganti** [comment]
I have added nextjs generator as a blueprint ensuring standard Wedaa integrations with keycloak and spring gateway. Please check it out and let me know if there are any changes.

**Reply by praveen-wedaa** [reply]
When can we plan to release this as part of WeDAA offering team ?
